scripts/vmpublish.rb

```diff
@@ -0,0 +1,80 @@
+require 'logger'
+require 'pathname'
+require 'trollop'
+require 'json'
+
+require_relative 'config'
+
+
+def find_imgfac_image(image)
```

Comment:
The files in /build/storage get cleaned up daily (if you follow our instruction to setup build machine).  If the steps are separate and `publish` can happen later, *.body files might be gone.